app/controllers/sessions_controller.rb

```diff
@@ -1,9 +1,28 @@
 # frozen_string_literal: true
 
 class SessionsController < Devise::SessionsController
+  prepend_before_action :log_signout, only: :destroy
   prepend_before_action :check_captcha, only: [:create] # Change this to be any actions you want to protect.
 
   private
+
+    def log_signout
+      ActiveSupport::Notifications.instrument "devise_action.action_signout", signout_params
```

Comment:
@radinreth move "devise_action.action_signout" to class constant variable to be used in activity log subscription